src/components/LoansByCategory/HelpmeChoose/HelpmeChooseRecommendations.vue

```diff
@@ -115,6 +124,25 @@ export default {
 	methods: {
 		selectLoan(evt) {
 			this.selectedLoan = evt;
+			let eventProperty = '';
+			switch (evt) {
+				case 0:
+					eventProperty = 'help-me-choose-loan-1';
```

Comment:
Do we want info on which specific loan was clicked? Like the ID? Or do we just care to know if one of the loans were clicked?

Reply:
I asked the same to my PM and said that just needing if one of the loans were clicked.
Thanks to ask @dyersituations